From 7749b481ce5d7e232b1f7da5e6b2c44816f51681 Mon Sep 17 00:00:00 2001
From: Jonas Gorski <jogo@openwrt.org>
Date: Sun, 19 Jan 2014 20:45:51 +0100
Subject: [PATCH 2/2] net: provide defines for _POLICY_FAILED until all code is
 updated

Upstream introduced ICMPV6_POLICY_FAIL for code 5 of destination
unreachable, conflicting with our name.

Add appropriate defines to allow our code to build with the new
name until we have updated our local patches for older kernels
and userspace packages.

Signed-off-by: Jonas Gorski <jogo@openwrt.org>
---
 include/uapi/linux/fib_rules.h | 2 ++
 include/uapi/linux/icmpv6.h    | 2 ++
 include/uapi/linux/rtnetlink.h | 2 ++
 3 files changed, 6 insertions(+)

--- a/include/uapi/linux/icmpv6.h
+++ b/include/uapi/linux/icmpv6.h
@@ -118,6 +118,8 @@ struct icmp6hdr {
 #define ICMPV6_POLICY_FAIL		5
 #define ICMPV6_REJECT_ROUTE		6
 
+#define ICMPV6_FAILED_POLICY		ICMPV6_POLICY_FAIL
+
 /*
  *	Codes for Time Exceeded
  */
